src/CodeGen_LLVM.cpp

```diff
@@ -283,7 +277,8 @@ CodeGen_LLVM *CodeGen_LLVM::new_for_target(const Target &target,
                                 Target::OpenCL,
                                 Target::OpenGL,
                                 Target::OpenGLCompute,
-                                Target::Metal})) {
+                                Target::Metal,
+                                Target::AMDGPUGFX900})) {
```

Comment:
The target should probably be AMDHSA since that is the target ABI for which code is being generated.